http_stubs/views.py

```diff
@@ -64,3 +64,14 @@ def dispatch(self, request: HttpRequest, *args, **kwargs) -> HttpResponse:
         for header_name, header_value in stub.resp_headers.items():
             response[header_name] = header_value
         return response
+
+    def remove_empty_headers(self, headers: Dict[str, str]) -> Dict[str, str]:
```

Comment:
Just asking: can't be empty headers significant? E.g. indicate there's something wrong on the sending side?